### ISSUE TITLE
Support for additional FFmpeg flags

### ISSUE THREAD

**Issue by Krzysztofz01** [AlexEidt/Vidio#4]
I thought about opening a pull request with a feature implementation I wanted to propose, but I think, first I'm gonna consult it via an issue.
I thought about the possibility of passing additional FFmpeg flags and parameters to `VideoWriter`. This could be achieved, for example, by adding a `map[string]string` or a `[]string` to the `Options` struct. This change wouldn't introduce any breaking-changes but may open new possibilities, more use-cases and would make the library generally more versatile.

**Comment by AlexEidt** [comment]
Hi Krzysztof, thanks for the suggestion. In early drafts of the project this was actually a feature I implemented. I did not end up including it because I wanted to abstract away ffmpeg. The user should focus on video I/O and not have to worry about being familiar with ffmpeg commands. `vidio` is a simple video I/O tool that happens to use FFmpeg in the backend. If users wish to use more complex ffmpeg commands, then calling `exec.Cmd` with the desired parameters would be better.
